articles/welcome.md

```diff
@@ -10,48 +10,71 @@ ms.topic: overview
 
 # Get started with the Quantum Development Kit (QDK)
 
-Welcome to the Microsoft Quantum Development Kit!  Here, you will find all the tools you need to learn about quantum programming with Q#.  To guide you in learning quantum computing with Q#, we provide this Getting Started Guide, a journey guide for both those of you who want to start coding quantum programs, and people who are not yet ready to start coding but want to learn more about Q# and quantum programming.
+Welcome to the Microsoft Quantum Development Kit!  
+The QDK contains all the tools you'll need to build your own quantum computing programs and experiments with Q#, a programming language used for expressing quantum algorithms. 
+Specifically, Q# allows you to write sub-programs that execute on an adjunct quantum processor under the control of a classical host program and computer. 
 
-## Questions about quantum computing
+To jump right in, you can head over to the [QDK installation guide](xref:microsoft.quantum.install).
+You'll then be guided through installing the Quantum Development Kit on Windows, Linux, or macOS machines so that you can write your own quantum programs.
```

Comment:
```suggestion
You'll then be guided through installing the Quantum Development Kit on Windows, Linux, or MacOS machines so that you can write your own quantum programs.
```